src/lib.rs

```diff
@@ -10,11 +10,10 @@ extern crate futures_cpupool;
 extern crate slog;
 extern crate slog_term;
 
-mod request;
-mod response;
-mod error;
-mod status;
-
+pub mod request;
+pub mod response;
+pub mod error;
+pub mod status;
```

Comment:
sorry, this was the wrong change. pub *use* not pub mod.

```rust
mod request;
pub use request::Request;
```

etc. this change is what enables `sparkles::request::Request`, but the interface should be `sparkles::Request`.